accountability_project/Models/spaces/api/api.py

```diff
@@ -219,4 +220,14 @@ def get_queryset(self):
         logger.info(f' Get all users from space with id: {space_id}')
         space = get_object_or_404(Space, id=space_id, members=self.request.user)
         logger.info(f' Retrieving all users for space with id: {space_id} and name: {space.name}')
-        return space.members.all()
\ No newline at end of file
+        return space.members.all()
+
+class CalendarAPIView(generics.ListAPIView):
+    serializer_class = RecurrentHabitSerializerToRead
+
+    def get_queryset(self):
+        space_id = self.kwargs.get('pk')
```

Comment:
- [ ] please add this logging below this line for better traceability:

`logger.info(f' Get all recurrent habits from space with id: {space_id}')`